docs/database.md

```diff
@@ -34,3 +34,41 @@ To modify the schema there are a few locations in the code we need to update.
 1. [Line 77 of this file contains the table creation statements](.github/workflows/terraform-deploy_reusable.yml) To modify column data types or add or remove columns, go there.
 2. The `PostgresDao.java` contains our queries for the database. Remember to update both the save and the fetch methods
 3. `DatabasePartnerMetadataStorage` is what calls our dao to perform the db operation, you will need to update the inputs here
+
+## Database Migrations
+
+We use [Liquibase](https://www.liquibase.com/download) to handle the migrations.  Our migration files are located in
+[`/etor/databaseMigrations`](/etor/databaseMigrations).
+
+### Adding Migrations
+
+Documentation on Liquibase can be found [here](https://docs.liquibase.com/).
+
+For any migrations needed for a backlog item, add a new file to [`/etor/databaseMigrations`](/etor/databaseMigrations)
+with the number of the story.  For example, `753.yml` for
```

Comment:
Should we organize them by story or by database object? I would think it would be challenging to find an issue if they are by story